src/ludamus/edges/rituals.py

```
@@ -0,0 +1,916 @@
+
+
+# Nothing reads what the agent said back: these calls are judged by what they
+# left in the worktree, which the step that follows reads out of git. So the
+# only answer worth returning is whether the agent was still standing.
+async def _ask(
+    prompt: str, *, opts: CodingOpts | None = None, key: str | None = None
+) -> Fallen | None:
+    session = Session.CONTINUE if key is not None else Session.NEW
+    try:
+        await coding(prompt, opts=opts, session=session, key=key)
+    except Exception as error:  # ruff: ignore [blind-except]
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)